stories/core/Table.stories.tsx

```diff
@@ -307,6 +312,119 @@ Sortable.args = {
   isSortable: true,
 };
 
+export const Filters: Story<TableProps> = (args) => {
```

Comment:
I plan to add more stories to show how to use manual filtering e.g. in server-side, maybe how to use custom filter function and writing custom filter component.